### ISSUE TITLE
php7

### ISSUE THREAD

**Issue by ipolo0000i** [raspiweather/raspiweather#9]
Don't work with php7!

**Comment by shreddedbacon** [comment]
Thanks for raising this issue. I haven't worked on this project properly for some time.

It may be some time before I get a chance to test this with PHP 7

**Comment by ipolo0000i** [comment]
Hey thank you for answer! I confirm it's work with php7 but it needs somes tweaks.

**Comment by shreddedbacon** [comment]
Yeah quick look at it, it looks like Nginx config needs to know the new socket path/url for PHP. I can't think of any of the actual PHP that would be the issue.

I am actually looking at re-writing the whole thing at some stage too.
